### ISSUE TITLE
'not installed' when installed?

### ISSUE THREAD

**Issue by crawft** [dpa99c/cordova-diagnostic-plugin#202]
Thanks for the help...

When I try to use this on a device I get this in the log:

```
Native: tried calling Diagnostic.isRemindersAuthorized, but the Diagnostic plugin is not installed.
Install the Diagnostic plugin: 'ionic plugin add cordova.plugins.diagnostic'
```

Device is definitely ready, I've tried removing and re-adding the plugin successfully, but still the same error at runtime - any ideas?

**Comment by dpa99c** [comment]
If the plugin is present in `plugins/cordova.plugins.diagnostic` folder, then it is installed in the project. 

The `Diagnostic` namespace refers to the [Ionic Native wrapper](https://github.com/driftyco/ionic-native/blob/master/src/%40ionic-native/plugins/diagnostic/index.ts) which is a component of [ionic-native](https://github.com/driftyco/ionic-native) and therefore any issues with it should be reported against that repo, as it is not part of this plugin.
